editor.planx.uk/src/@planx/components/List/utils.tsx

```diff
@@ -1,24 +1,40 @@
-import { QuestionField, Schema, UserResponse } from "./model";
+import React from "react";
+
+import { Field, UserResponse } from "./model";
 
 /**
- * In the case of "question" fields, ensure the displayed value reflects option "text", rather than "val" as recorded in passport
+ * In the case of "question" and "checklist" fields, ensure the displayed value reflects option "text", rather than "val" as recorded in passport
  * @param value - the `val` or `text` of an Option defined in the schema's fields
- * @param schema - the Schema object
- * @returns string - the `text` for the given value `val`, or the original value
+ * @param field - the Field object
+ * @returns string | React.JSX.Element - the `text` for the given value `val`, or the original value
  */
-export function formatSchemaDisplayValue(value: string, schema: Schema) {
-  const questionFields = schema.fields.filter(
-    (field) => field.type === "question",
-  ) as QuestionField[];
-  const matchingField = questionFields?.find((field) =>
-    field.data.options.some((option) => option.data.val === value),
-  );
-  const matchingOption = matchingField?.data.options.find(
-    (option) => option.data.val === value,
-  );
-
-  // If we found a "val" match, return its text, else just return the value as passed in
-  return matchingOption?.data?.text || value;
+export function formatSchemaDisplayValue(
+  value: string | string[],
+  field: Field,
+) {
+  switch (field.type) {
+    case "number":
+    case "text":
+      return value;
+    case "checklist": {
+      const matchingOptions = field.data.options.filter((option) =>
+        (value as string[]).includes(option.id),
+      );
+      return (
+        <ul>
+          {matchingOptions.map((option) => (
+            <li key={option.id}>{option.data.text}</li>
+          ))}
+        </ul>
```

Comment:
nit: can we slightly adjust padding/margin here so bullet points are aligned to right text rather than indented? 
![Screenshot from 2024-06-27 12-43-26](https://github.com/theopensystemslab/planx-new/assets/5132349/a6d0a634-03bf-477b-a09a-635a34808d8c)

I think that would be more consistent with existing SummaryList/Review page spacing